src/app/find-help/reply-offer/reply-offer.component.ts

```diff
@@ -29,7 +29,12 @@ export class ReplyOfferComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
-    this.data.tosApproved = false;
+    if (this.authService.isLoggedIn()) {
+      this.data.tosApproved = true;
+    } else {
+      this.data.tosApproved = false;
```

Comment:
this.data.tosApproved = this.authService.isLoggedIn()
Could work?

Reply:
poprawione